jbpm-wb-common/jbpm-wb-common-client/src/test/java/org/jbpm/workbench/common/client/list/AbstractMultiGridViewTest.java

```diff
@@ -232,4 +236,81 @@ public void testControlBulkOperationsDisabled() {
         getView().controlBulkOperations(extendedPagedTable);
         verify(mockButton).setEnabled(false);
     }
-}
\ No newline at end of file
+
+    @Test
+    public void testReloadColumnSortListWhenInitListTable() {
+        ListTable<T> listTable = mock(ListTable.class);
+        when(getView().getListGrid()).thenReturn(listTable);
+        GridPreferencesStore gridPreferencesStore = mock(GridPreferencesStore.class);
+        when(listTable.getGridPreferencesStore()).thenReturn(gridPreferencesStore);
+        GridSortedColumnPreference gridSortedColumnPreference = new GridSortedColumnPreference("test",
+                                                                                               true);
+        when(gridPreferencesStore.getGridSortedColumnPreference()).thenReturn(gridSortedColumnPreference);
+        ColumnSortList columnSortList = getColumnSortList("startDate",
+                                                          null);
```

Comment:
These newline are not really necessary, right? I've seen in other PRs that we try to prevent/remove these. (multiple occurrences in the newly added test methods.)

Reply:
I'm not sure what your mean ?

Reply:
Just a matter of code formatting. Splitting the line across two lines in this case doesn't seem to be necessary (the line above is even longer, but not split)

Reply:
it is kie code style.

Reply:
@cristianonicolai how is it with code style now? :-) We've had similar discussion in https://github.com/kiegroup/jbpm-wb/pull/1284. Could you please share your view?

Reply:
breaking line should be only applied when necessary, its not mandatory any more. I would say to only do it if there is no space left in the current line. I agree that for this line it seems unnecessary.